CMakeLists.txt

```diff
@@ -3,10 +3,15 @@ project(ArborX CXX)
 
 set(CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake ${CMAKE_MODULE_PATH})
 find_package(Kokkos REQUIRED)
-find_package(MPI REQUIRED)
 
 add_library(ArborX INTERFACE)
-target_link_libraries(ArborX INTERFACE Kokkos::Kokkos MPI::MPI_CXX)
+target_link_libraries(ArborX INTERFACE Kokkos::Kokkos)
+
+option(ArborX_ENABLE_MPI "Enable MPI support" OFF)
```

Comment:
Will need to discuss defaults

Reply:
Unless we try to autodetect, I would default to minimal dependencies but don't have a strong opinion here.